pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/kit/ClinicalKitDto.java

```diff
@@ -101,25 +101,25 @@ public void setGender(String genderString) {
     }
 
     public void setNecessaryParticipantDataToClinicalKit(String ddpParticipantId, DDPInstance ddpInstance) {
+        logger.info("Setting PHI to clinical kit's accessioning data for {}", ddpParticipantId);
         Optional<ElasticSearchParticipantDto> maybeParticipantESDataByParticipantId =
                 ElasticSearchUtil.getParticipantESDataByParticipantId(ddpInstance.getParticipantIndexES(), ddpParticipantId);
         if (maybeParticipantESDataByParticipantId.isEmpty()) {
             throw new RuntimeException("Participant ES Data is not found for " + ddpParticipantId);
         }
-
+        ElasticSearchParticipantDto elasticSearchParticipantDto = maybeParticipantESDataByParticipantId.get();
         try {
-            this.setDateOfBirth(maybeParticipantESDataByParticipantId.get().getDsm().map(Dsm::getDateOfBirth).orElse(""));
-            this.setFirstName(maybeParticipantESDataByParticipantId.get().getProfile().map(Profile::getFirstName).orElse(""));
-            this.setLastName(maybeParticipantESDataByParticipantId.get().getProfile().map(Profile::getLastName).orElse(""));
-            this.setGender(getParticipantGender(maybeParticipantESDataByParticipantId.get(), ddpInstance.getName()));
-            String shortId = maybeParticipantESDataByParticipantId.get().getProfile().map(Profile::getHruid).orElse("");
+            this.setDateOfBirth(elasticSearchParticipantDto.getDsm().map(Dsm::getDateOfBirth).orElse(""));
+            this.setFirstName(elasticSearchParticipantDto.getProfile().map(Profile::getFirstName).orElse(""));
+            this.setLastName(elasticSearchParticipantDto.getProfile().map(Profile::getLastName).orElse(""));
+            this.setGender(getParticipantGender(elasticSearchParticipantDto, ddpInstance.getName()));
```

Comment:
I am not sure what value is in the name but we are setting "name" into "Gender" ?

Reply:
the name is the study name, like `cmi-lms` in this case